### PR TITLE
Bromley Waste Phase 2

### PR DESCRIPTION
**Pull request by struan**
Updates required for Garden Waste subscriptions

### CODE REVIEW

File: perllib/Open311/Endpoint/Integration/Echo.pm [hunk 1]
```diff
@@ -90,6 +90,20 @@ sub get_integration {
...
 # or any given defaults
 sub check_for_data_value {
+    my ($self, $name, $args, $request, $parent_name) = @_;
+    my ($value, $full_name);
+    $full_name = $parent_name . '_' . $name if $parent_name;
+
+    if ($full_name) {
+        $value = $self->_get_data_value($full_name, $args, $request);
+    }
+    unless ($value) {
```
Comment (dracos) [inline perllib/Open311/Endpoint/Integration/Echo.pm:100]: Not sure if this is needed but could value ever be desirably 0? should it be checking defined?

**Review by dracos — APPROVED** [review]
Looks good! One question I wasn't sure about re possible values of 0.

**Review by dracos — APPROVED** [review]
0️⃣

**Comment by struan** [comment]
Closing as identical to new bromley-waste-phase2-reviewed branch

**Pull request closed without merging**